source/plugin/economy/using.rst

```diff
@@ -0,0 +1,71 @@
+=====================
+Using the Economy API
+=====================
+
+The Economy API unifies all economy plugins under one API. This means any plugin using the Economy API
+will be compatible with all economy plugins that implement the API. This page guides you through the steps of using
+the Economy API in your own plugin.
+
+Loading the EconomyService
+==========================
+
+In order to utilize the Economy API, you must first load the EconomyService class:
+
+1. Listen to the ``ChangeServiceProviderEvent`` in order to grab an instance of the EconomyService when it is registered.
+
+#. When the event is fired, check if the service added was the ``EconomyService``. If the it was the ``EconomyService``,
```

Comment:
Wording is funny. Also it would probably be better to use ``Sponge.getGame().getServiceManager().provide(EconomyService.class)``. If the providing plugin honors the statements made for the plugin lifecycle it will be safe to get the economy service in the Initialization phase, or PostInit at the latest.